### ISSUE TITLE
Custom optimiser based on pre-trained model

### ISSUE THREAD

**Issue by Mehdzor** [tensorflow/tensorflow#17876]
Hello. This might be a support ticket or a feature request.

I'm trying to make an optimiser based on output of a pretrained model. Running a model requires an open session in tensorflow but there is nothing like it inside OpKernel where the existing model should be called. 

How to correctly call a model inside an opkernel? Is it possible to access already compiled functionality  of this kind? 

This feature would be very handy when it comes to complex loss functions. I'm able to write it on my own and wrap it in a PR when my questions are resolved.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Mehdzor** [comment]
Have I written custom code: Not yet
OS Platform and Distribution: OSX 10.13
TensorFlow installed from: Source
TensorFlow version: 1.5
Bazel version: 0.11.0
CUDA/cuDNN version: 9.0 / 7.0
GPU model and memory: 2 * GTX 1070, 8gb
Exact command to reproduce: N/A

**Comment by rohan100jain** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
